src/main/java/school/lemon/changerequest/java/introduction/pr1/HammingDistance.java

```diff
@@ -1,7 +1,20 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
+
 public class HammingDistance {
     public static int compute(String a, String a1) {
-        return -1;
+        if (a.length() != a1.length()) return -1;
+        int count = 0;
+        for (int i = 0, j = 0; i < a.length(); i++, j++) {
```

Comment:
Now you are 100% sure than length is the same - you don't need additional index variable - just reuse `i` or `j`